tutorials/export/exporting_for_visionos.rst

```diff
@@ -0,0 +1,19 @@
+.. _doc_exporting_for_visionos:
+
+Exporting for visionOS
+======================
+
+.. seealso::
+
+    This page describes how to export a Godot project to visionOS.
+    If you're looking to compile export template binaries from source instead,
+    read :ref:`doc_compiling_for_visionos`.
+
+Exporting instructions for visionOS are identical to :ref:`doc_compiling_for_ios`,
+except you should add a **visionOS** export preset instead of **iOS**.
+See the linked page for details.
```

Comment:
```suggestion
Exporting instructions for visionOS are for now identical to :ref:`doc_compiling_for_ios`,
except you should add a **visionOS** export preset instead of **iOS**.
See the linked page for details.
```